Nation/Various/SwirlingTheAbyss.cs

```diff
@@ -0,0 +1,64 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Nation/CoreNation.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Story/Nation/Originul.cs
+//cs_include Scripts/Story/Nation/Fiendshard.cs
+using RBot;
+
+public class SwirlingTheAbyss
+{
+    public ScriptInterface Bot => ScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new CoreFarms();
+    public CoreNation Nation = new();
+    public Fiendshard_Story Fiendshard = new Fiendshard_Story();
+
+    public void ScriptMain(ScriptInterface bot)
+    {
+        Core.BankingBlackList.AddRange(Nation.bagDrops);
+        Core.BankingBlackList.AddRange(Rewards);
+        Core.SetOptions();
+
+        STA();
+
+        Core.SetOptions(false);
+    }
+
+    public readonly string[] Rewards =
+ {
+        "ArchFiend Deathlord Armet",
+        "ArchFiend Deathlord Horned Armet",
+        "ArchFiend Deathlord Helmet",
+        "ArchFiend Deathlord Mask",
+        "ArchFiend Deathlord Locks",
+        "ArchFiend Deathlord Shag",
+        "DeathLord Cloak of Nulgath",
+        "Void Soul of Nulgath",
+        "DeathLord Spines of Nulgath",
+        "Deathless Wings of Nulgath",
+    };
+    public void STA()
+    {
+        Fiendshard.Fiendshard_Questline();
+        Core.AddDrop(Nation.bagDrops);
+        Core.AddDrop(Rewards);
+
+        int i = 1;
+        while (!Bot.ShouldExit() && !Core.CheckInventory(Rewards, toInv: false))
+        {
+            Nation.FarmBloodGem(10);
+            Nation.FarmUni13(3);
+            Nation.SwindleBulk(75);
+            Nation.FarmDarkCrystalShard(50);
+            Nation.FarmGemofNulgath(50);
+            Nation.FarmVoucher(true);
+
+            Core.EnsureAccept(7899);
+            Core.EnsureCompleteChoose(7899);
+            Core.Logger($"Completed x{i++}");

```

Comment:
look at the quest, the rewards are all ac items, can you add a "Core.ToBank(Rewards);" above the logger so it'll bank it after each one.. for inventory purposes >.> as when dealing with nation stuff basic non-whaled baggs will cry :P